k8s/src/main/scala/io/buoyant/k8s/istio/PollingApiClient.scala

```diff
@@ -16,6 +16,9 @@ abstract class PollingApiClient(client: Service[Request, Response]) {
   protected val log = Logger()
   protected val mapper = Parser.jsonObjectMapper(Nil)
   mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+  registerModules(mapper)
+
+  def registerModules(mapper: ObjectMapper) {}
```

Comment:
I think scala gets annoyed if you don't put in the Unit return type and the `=`.